german/suchen/suchen.md

```diff
@@ -2,6 +2,6 @@ Ich suche im Boot herum und entdecke einen alten Ordner, den ich als Paddel benu
 
 Was mache ich jetzt?
 
-1. Ich fange an damit zu paddeln.] (../paddeln/paddeln.md)
```

Comment:
I just repaired a broken link by adding the second bracket which was missing before.